tools/wavetool.py

```diff
@@ -0,0 +1,150 @@
+#! /usr/bin/python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+"""
+A Tool to Generate and Manipulate Wave Files
```

Comment:
Also this is rather brief description. Is there somewhere a graph of test system? If not, could even add an ASCII graph here for description what the test does.

Reply:
For DSM case, the tool only do binary comparison. There is sure case description in check-smart-amplifier.sh